docusaurus/docs/React/guides/sdk-state-management.mdx

```diff
@@ -152,45 +152,45 @@ Selectors are functions provided by integrators that run whenever state object c
 
 #### Rules of Selectors
 
-1. Selectors should return array of data sorted by their "change factor"; meaning values that change often should come first for the best performance.
+1. Selectors should return a named object.
 
 ```ts
-const selector = (nextValue: ThreadManagerState) => [
-  nextValue.unreadThreadsCount, // <-- changes often
-  nextValue.active, // <-- changes less often
-  nextvalue.lastConnectionDownAt, // <-- changes rarely
-];
+const selector = (nextValue: ThreadManagerState) => ({
+  unreadThreadsCount: nextValue.unreadThreadsCount, // <-- changes often
+  active: nextValue.active, // <-- changes less often
+  lastConnectionDownAt: nextvalue.lastConnectionDownAt, // <-- changes rarely
+});
```

Comment:
```suggestion
const selector = (nextValue: ThreadManagerState) => ({
  unreadThreadsCount: nextValue.unreadThreadsCount,
  active: nextValue.active,
  lastConnectionDownAt: nextvalue.lastConnectionDownAt,
});
```

Do we have guarantees for consistent `for-in` iteration over objects? If no, `changes` frequency becomes irrelevant.